.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,17 @@
+name: pull_request
+
+on:
+  pull_request:
+    branches: [ master ]
+env:
+  DEVELOPER_DIR: /Applications/Xcode_11.4.app/Contents/Developer
```

Comment:
Set to use Xcode 11.4

Reply:
How does this DEVELOPER_DIR work? Is that referencing some virtual machine being run by `Github Actions`?

Reply:
This is not very well documented at all, so I had to do some digging too.  Sorry for not explaining this already!

I found these posts: 
https://www.jessesquires.com/blog/selecting-an-xcode-version-on-github-ci/
https://github.community/t5/GitHub-Actions/Selecting-an-Xcode-version/td-p/31105

I would imagine they will add a more explicit way to set Xcode version, but this is how it's apparently supposed to be done today.

Reply:
Updated [here](https://github.com/covid19risk/covidwatch-ios/pull/20/commits/b11773639a5e75a6fb5381ce5e3cb0f94ccb0649).